junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/ParameterExtensionTest.java

```diff
@@ -39,59 +38,52 @@
 import static org.awaitility.Awaitility.await;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertInstanceOf;
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
-import static org.junit.jupiter.api.Assertions.assertNotSame;
-import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertSame;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(KafkaClusterExtension.class)
-public class ParameterExtensionTest extends AbstractExtensionTest {
+class ParameterExtensionTest extends AbstractExtensionTest {
 
     private static final Duration CLUSTER_FORMATION_TIMEOUT = Duration.ofSeconds(10);
     private static final String CONSUMER_GROUP = "mygroup";
     private static final String TRANSACTIONAL_ID = "mytxn1";
     private static final String CLIENT_ID = "myclientid";
 
     @Test
-    public void clusterParameter(@BrokerCluster(numBrokers = 2) KafkaCluster cluster)
-            throws ExecutionException, InterruptedException {
-        await().atMost(CLUSTER_FORMATION_TIMEOUT).untilAsserted(() -> assertEquals(2, describeCluster(cluster.getKafkaClientConfiguration()).nodes().get().size()));
-        var dc = describeCluster(cluster.getKafkaClientConfiguration());
-        assertEquals(cluster.getClusterId(), dc.clusterId().get());
-        assertInstanceOf(InVMKafkaCluster.class, cluster);
+    void clusterParameter(@BrokerCluster(numBrokers = 2) KafkaCluster cluster) throws Exception {
+        assertThat(cluster).isInstanceOf(InVMKafkaCluster.class);
+        assertClusterIdAndSize(cluster, 2);
     }
 
     @Test
-    public void brokerConfigs(@BrokerConfig(name = "compression.type", value = "zstd") @BrokerConfig(name = "delete.topic.enable", value = "false") KafkaCluster clusterWithConfigs,
-                              Admin admin)
+    void brokerConfigs(@BrokerConfig(name = "compression.type", value = "zstd") @BrokerConfig(name = "delete.topic.enable", value = "false") KafkaCluster clusterWithConfigs,
+                       Admin admin)
             throws Exception {
         assertSameCluster(clusterWithConfigs, admin);
-        ConfigResource resource = new ConfigResource(ConfigResource.Type.BROKER, "0");
+        var resource = new ConfigResource(ConfigResource.Type.BROKER, "0");
+
         var configs = admin.describeConfigs(List.of(resource)).all().get().get(resource);
-        assertEquals("zstd", configs.get("compression.type").value());
-        assertEquals("false", configs.get("delete.topic.enable").value());
+        assertThat(configs.get("compression.type")).extracting(ConfigEntry::value).isEqualTo("zstd");
+        assertThat(configs.get("delete.topic.enable")).extracting(ConfigEntry::value).isEqualTo("false");
```

Comment:
Don't get me wrong I think the switch to assertJ is a good one, but that line still makes me a little sad. It would be good to have a custom assertion or something that understood `configs` is effectively a map...

Reply:
Fixed.